usr/share/lib/img_proof/tests/SLES/SAP/test_sles_sap_hardened.py

```diff
@@ -0,0 +1,60 @@
+import os
+import pytest
+
+
+@pytest.mark.skipinbeta
+def test_sles_hardened(host, get_release_value):
+    variant = get_release_value('VARIANT_ID') or ''
+
+    if "hardened" not in variant:
```

Comment:
Nitpick: the use of double quotes is inconsistent with the rest of the project which uses single quotes by default.

Reply:
> Nitpick: the use of double quotes is inconsistent with the rest of the project which uses single quotes by default.

Shouldn't this be changed eventually?  PEP enforcers like Black use double-quotes by default.

Reply:
Well "nitpick" is only a suggestion so not PR blocking. Mainly was a comment about consistency. Since single and double quotes are interchangeable in python. And there's no plans to change all code to match an opinionated choice made by a code formatter 🤷🏼 .